### PR TITLE
restful routes + state update

### PR DESCRIPTION
**Pull request by charneff**
Added language and table for restful routes. Also added a line about requiring state update after a post request. Please feel free to reword if confusing! Thank you for your help!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,14 +24,49 @@ listed in these instructions.
    organized.
 3. There should be at least 3 client-side routes using [React
    Router][react-router]. Be sure to include a nav bar or other UI element that
-   allows users to navigate between routes.
+   allows users to navigate between routes. Use RESTful routing convention where
```
Comment (lizbur10) [inline README.md]: Edit: "...routing conventions"

File: README.md [hunk 2]
```diff
@@ -24,14 +24,49 @@ listed in these instructions.
...
+     }
+
+     fetch('http://localhost:3000/movies', configObj)
+      .then(res => res.json())
+      .then(data => addMovie(data)) //THIS STATE UPDATE IS REQUIRED!!!
+      // clear form
+
+     ```
+     Do NOT rely solely on the useEffect of another component to load your new data
+     with another GET fetch request.
```
Comment (lizbur10) [inline README.md]: I'd like to make this sound a little more instructional/education-y if we can. What about something like: "Remember: responsibility for re-rendering the page to display the updated list of movies should belong to the `addMovie` function; depending on a subsequent action to load the new data is not best practice." 

Feel free to edit if that doesn't fully get at what you're going for here.

**Review by lizbur10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (charneff) [reply README.md]: PERFECT!

File: README.md (hunk 2 above)
Comment (charneff) [reply README.md]: That sounds awesome! Thank you, Liz!!!

**Comment by charneff** [comment]
I am totally good with these proposed changes! Thank you so much, Liz!

**Pull request merged**